contracts/mocks/Mock0xExchange.sol

```diff
@@ -29,11 +29,22 @@ contract Mock0xExchange {
         IZeroXExchange.Order memory _order,
         uint256 _takerAssetFillAmount,
         bytes memory _signature
-    ) external payable returns (IZeroXExchange.FillResults memory fillResults) {
+    ) public payable returns (IZeroXExchange.FillResults memory fillResults) {
         takerAmount = _order.takerAssetAmount;
         makerAmount = _order.makerAssetAmount;
         signature = _signature;
         fillAmount = _takerAssetFillAmount;
         return IZeroXExchange.FillResults(0, 0, 0, 0, 0);
     }
+
+    function batchFillOrders(
```

Comment:
`batchFillOrders` returns `(LibFillResults.FillResults[] memory fillResults)`, I think we should mock the same returns result and return back an array with the same length as orders.